Controller/AbstractAction.php

```diff
@@ -64,6 +64,11 @@ abstract class AbstractAction extends \Magento\Framework\App\Action\Action
      */
     public $orderSender;
 
+    /**
```

Comment:
Please place it in the same order as in constructor (between orderFactory and configSettings)